### DISCUSSION TITLE
Feature overview

### DISCUSSION THREAD

**Discussion by DL4OCE** [dl9rdz/rdz_ttgo_sonde#312] [Q&A] [answered]
Hi, 

I'm looking for a fork that includes an upload-to-radiosondy.info feature. 

Is there an overview of all the forks, or do I have to browse them all...?

Best 73 de Arne, DL4OCE   ..._._

**Comment by dl9rdz** [comment, marked as answer]
Why looking for a fork? The existing APRS upload features can be used directly with radiosondy
 It supports sonde position and fixed receiver position. 
Mobile receiver position (from T-Beam GPS or equivalent) is on the TODO list (but I don't know of any fork that implements that already either :)

**Reply by DL4OCE** [reply]
I'm so sorry I didn't find that information neither in the documentation or in the source code, it's probably been too early in the morning :-D

Thank you very much!

**Reply by dl9rdz** [reply]
I am sure that the documentation can be improved :)

But its here: https://github.com/dl9rdz/rdz_ttgo_sonde/wiki/Data-feed-configuration "APRS parameters (for example, for radiosondy.info)"
